src/nrniv/nrnbbcore_write/utils/corenrn_utils.cpp

```diff
@@ -0,0 +1,221 @@
+#include "corenrn_utils.h"
+#include "nrnbbcore_write/callbacks/bbcore_callbacks.h"
+
+#include "nrnconf.h"
+#include <cstdlib>
+#include "nrndae_c.h"
+#include "section.h"
+#include "hocdec.h"
+#include "nrnsection_mapping.h"
+#include "vrecitem.h" // for nrnbbcore_vecplay_write
+#include "parse.h"
+#include <string>
+#include <unistd.h>
+#include <algorithm>
+#include <cerrno>
```

Comment:
`nitpick` : can we separate / group standard headers vs nrn headers?

Reply:
Not hard, but I prefer to do with the refactoring, since those will change anyway.